### DISCUSSION TITLE
Sending a POST message

### DISCUSSION THREAD

**Discussion by DJZuher** [savonet/liquidsoap#2329] [Q&A]
(Google translator)

Hello. Please help with recipe
Sending a POST message via http containing the filename when starting each track. It's the filename, not the ID
Is it possible to do this with liquidsoap?
I want to organize on the site "who is on the air", as well as counting the number of tracks played by comparing file names with entries in the database using PHP. 
thanks in advance

p.s. v.1.4.3

**Comment by toots** [comment]
Hi. Definitely possible. The documentation for the operator is here: https://www.liquidsoap.info/doc-1.4.4/reference.html#http.post
